sqlmesh/core/context.py

```diff
@@ -218,20 +217,16 @@ def __init__(
         self._hooks: UniqueKeyDict[str, hook] = UniqueKeyDict("hooks")
 
         self.path, self.config = t.cast(t.Tuple[Path, Config], next(iter(self.configs.items())))
-        self._scheduler = self.config.scheduler
-        self.connection = connection
-        self.test_connection = test_connection
+        self.gateway = gateway
```

Comment:
just so you're aware of the implications on line 219, and also with self.gateway, we're making t he assumption that multi repo works with one gateway